memote/suite/tests/test_biomass.py

```diff
@@ -346,8 +351,9 @@ def test_fast_growth_default(model, reaction_id):
 
     """
     ann = test_fast_growth_default.annotation
-    ann["data"][reaction_id] = helpers.run_fba(model, reaction_id) > 2.81
-    ann["metric"][reaction_id] = 1.0  # Placeholder value.
+    test_outcome = helpers.run_fba(model, reaction_id) > 2.81
+    ann["data"][reaction_id] = test_outcome
+    ann["metric"] = 1.0 - float(test_outcome)
```

Comment:
Same here. Your metric would be overwritten with each reaction that is tested. The final metric won't be representative of the entire test but just of the final iteration or not?